src/models/exclude_filter.rs

```diff
@@ -15,8 +15,8 @@ use getset::Getters;
 use serde_derive::Deserialize;
 
```

Comment:
On the same lines, let's rename `constraints.rs` to `filters.rs`

Reply:
It's named `constraint.rs`. So I think it should be renamed to `filter.rs`.
Or shall I keep it as `filters.rs`?

Reply:
`filter.rs`

Reply:
Updated accordingly